cmd/ocsp-updater/main.go

```diff
@@ -422,6 +440,13 @@ func (updater *OCSPUpdater) oldOCSPResponsesTick(ctx context.Context, batchSize
 		return err
 	}
 
+	for _, s := range statuses {
```

Comment:
> c) certificates that haven't expired, but have a stale OCSP response

If "haven't expired" is determined by the `isExpired` field, then we will occasionally find certificates through this query that are recently expired. We can use that opportunity to set `isExpired = true`.